packages/core/plugin/src/withSentryAndroidGradlePlugin.ts

```diff
@@ -0,0 +1,96 @@
+import { withAppBuildGradle, withProjectBuildGradle } from '@expo/config-plugins';
+
+export interface SentryAndroidGradlePluginOptions {
+  sentryAndroidGradlePluginVersion?: string;
+  includeProguardMapping?: boolean;
+  dexguardEnabled?: boolean;
+  autoUploadNativeSymbols?: boolean;
+  autoUploadProguardMapping?: boolean;
+  uploadNativeSymbols?: boolean;
+  includeNativeSources?: boolean;
+  includeSourceContext?: boolean;
+  autoInstallationEnabled?: boolean;
```

Comment:
Let's not expose this, the auto install should be always disabled for RN.

Reply:
Removed with c153ed7f4e0c34eec32fd70933bca7e834c7e9db